pages/pateter.tsx

```diff
@@ -5,6 +5,7 @@ import Header from "../components/Header"
 import ReactPageScroller from 'react-page-scroller';
 import { useState } from "react";
 import PageInfo from "../components/PageInfo";
+import { doesNotThrow } from "assert";
```

Comment:
varför la du in denna? används den forf?

Reply:
Oj det var nog en auto completegrej som skedde när jag skrev något helt annat, och så autoimportade den autocompletegrejen in det

Reply:
jag kan nog fixa den då